build.gradle

```diff
@@ -312,7 +312,10 @@ project(':forge') {
                 source sourceSets.main
                 source sourceSets.userdev
                 
-                args '--mod', 'forge', '--all', '--output', rootProject.file('src/generated/resources/')
+                args '--mod', 'forge', '--all', '--output', rootProject.file('src/generated/resources/'), '--validate',
```

Comment:
Should these still be here?

Reply:
Unfortunately yes. I need to tell MC where to look for existing model/texture files (and enable validation), it doesn't automatically use the resource directories from the other run configs (which makes sense, we wouldn't want to count a previously generated file as preexisting). The second line is, as I noted, not ideal, but I don't know enough about (Forge)Gradle to implement a better solution.